test/routes/src/server/routes/t_github.js

```diff
@@ -19,6 +19,157 @@ describe('The GitHub API endpoint', () => {
   });
   app.use(github);
 
+  describe('get snapcraft.yaml route', () => {
```

Comment:
I guess we don't use out `/github/snapcraft-yaml/:owner/:name` route on client side anymore, but it was quite a nice way to test `internalGetSnapcraftYaml` in separation from more complicated functionality that uses it.